aip_x1_launch/launch/velodyne_node_container.launch.py

```diff
@@ -26,6 +26,8 @@
 
 def get_vehicle_info(context):
     gp = context.launch_configurations.get("ros_params", {})
```

Comment:
Same as https://github.com/tier4/autoware_launch/pull/328/files#r881396879.
It's better to keep a comment.

Reply:
I added comment https://github.com/tier4/aip_launcher/pull/58/commits/2c81daf4bfbaf5d533906942fa7dcc2c09703f90.